mcp/tools/helm.py

```
@@ -318,3 +318,50 @@ async def helm_search_repo(
+    
+    if values:
+        try:
+            with tempfile.NamedTemporaryFile(mode="w", suffix=".yaml", delete=False) as f:
+                f.write(values)
+                values_file = f.name
+
+            cmd += f" -f {values_file}"
+            result = await run_helm_command(cmd)
+
+            os.unlink(values_file)
+            return result
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)